test/OptionSetTest.php

```diff
@@ -973,6 +974,23 @@ public function testWithQuality()
         $this->assertEquals(70, $os->quality());
     }
 
+    public function testSetUnsafe(): void
+    {
+        $os = new OptionSet();
+        $optionName = 'zoom';
+        $args = ['2', '2'];
+
+        $os->setUnsafe($optionName, ...$args);
+
+        $reflectionClass = new ReflectionClass(OptionSet::class);
```

Comment:
I'm not a big fan of white-box approach in unit tests, but I appreciate very much your input and your commitment to improve this small library, and after all, this test code is of a good quality.